ui/app/AppLayouts/Chat/panels/communities/CommunityPermissionsSettingsPanel.qml

```diff
@@ -2,10 +2,14 @@ import QtQuick 2.14
 
 import "../../layouts"
 import "../../views/communities"
+import "../../stores"
 
 SettingsPageLayout {
     id: root
 
+    property var store: CommunitiesStore {}
```

Comment:
I think that in the future only this component should have access to store and operate on store. E.g `CommunityNewPermissionView` probably should not access store directly and be provided with only necessary model/props.

Reply:
It's a major discussion pending to be had for this topic, so I'll keep it as it is. Thanks for pointing it as well.